.github/workflows/bootstrap.yml

```diff
@@ -1,5 +1,10 @@
 name: Bootstrap
 
+# See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency.
+concurrency:
+  group: ${{ github.head_ref }}-${{ github.workflow }}
```

Comment:
why head_ref and not ref?

Reply:
Good point `head_ref` is only defined for pull requests and workflows are triggered for direct commits too. 
The consequence is that, for consecutive direct and merge commits in master (and 3.6, 3.4 etc), they will be cancelled but the last one.
So we will lose the status of those intermmediate commits. 
There is a small probability of a green pr breaking master cause we are not enforcing the pr is up to date with master before merging it. Also there is a even small probability of merge another pr just after the offending one and we will not know the real cause is the intemmediate and no the final one.
Will change asap